### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -2313,6 +2289,19 @@ SOFTWARE.
...
+## react-native-redash
+
+This product contains 'react-native-redash' by William Candillon.
+
+The React Native Reanimated and Gesture Handler Toolbelt.
+
+* HOMEPAGE:
+  * https://github.com/wcandillon/react-native-redash
+
+* LICENSE: MIT?
```
Comment (amyblais) [inline NOTICE.txt:2301]: I assume this is MIT? I couldn't find the license details on the repo

Comment (enahum) [reply NOTICE.txt:2301]: yes, it should be MIT, I've created an issue in the repo to see if the creator can add a license file

**Review by enahum — APPROVED** [review]

**Comment by enahum** [comment]
@amyblais license has been added in the repo

**Comment by amyblais** [comment]
@enahum Sorry, I don't have merge permissions here.

**Comment by enahum** [comment]
@amyblais i know you don't but can you ipdate the NOTICES?

**Review by enahum — APPROVED** [review]

**Pull request merged**